src/Components/MainMenu/withTitles.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../Styles/foundation/_variables.scss';
+
+.with-titles {
```

Comment:
1) I don't love this class name as it's not just titles it can be anything. It's just another level of nesting
2) Is there any way to dynamically apply this class based on the data passed in rather than making the consumer know about it?

Reply:
Doh :( I missed the fact that we can't import the variables outside of the library. 
I'll add a dynamic class update that will be triggered if the data goes another level deep.